fastlane/Fastfile

```diff
@@ -288,6 +288,49 @@ private_lane :emerge_upload do |options|
   )
 end
 
+lane :sync_xcodeproj_references do
+  Dir.chdir('..') do
+    project = Xcodeproj::Project.open(xcode_project)
+
+    untracked_files = sh('git status -s', log: false).split("\n").map(&:strip)
+    deleted_files = untracked_files.select { |f| f.start_with?('D ') && f.end_with?('.swift') }.map { |f| f.split.drop(1).join }
+    new_files = untracked_files.reject { |f| f.start_with?('M ', 'D ') }.map { |f| f.split.drop(1).join }
```

Comment:
selecting known prefixes is safer (?? and A)

Reply:
Updated the PR, check it out